### PR TITLE
fix cassandra&rocksdb&scylladb code checkstyle

### PR DESCRIPTION
**Pull request by z7658329**
fix cassandra&rocksdb&scylladb code checkstyle

### CODE REVIEW

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTables.java [hunk 1]
```diff
@@ -281,13 +281,13 @@ protected BackendColumnIterator queryByCond(Session session,
...
                     case GTE:
-                        minEq = true;
                     case GT:
+                        minEq = true;
                         min = (Id) r.value();
                         break;
                     case LTE:
-                        maxEq = true;
                     case LT:
+                        maxEq = true;
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTables.java:290]: changed the code logic

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTables.java [hunk 2]
```diff
@@ -281,13 +281,13 @@ protected BackendColumnIterator queryByCond(Session session,
                         prefix = (Id) r.value();
                         break;
                     case GTE:
-                        minEq = true;
                     case GT:
+                        minEq = true;
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBTables.java:285]: changed the code logic

**Review by javeme — CHANGES_REQUESTED** [review]
Still need to fix code logic changes

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java [hunk 3]
```diff
@@ -1058,14 +1059,14 @@ private boolean match(int expected) {
...
-                               (this.iter.isValid() ? "" : " - No data"));
+            LOG.info(">>>> scan from " + this.table + ": "  +
+                    (this.keyBegin == null ? "*" :
+                            StringEncoding.format(this.keyBegin)) +
+                    (this.iter.isValid() ? "" : " - No data"));
             for (; this.iter.isValid(); this.iter.next()) {
-                System.out.println(String.format("%s=%s",
-                                   StringEncoding.format(this.iter.key()),
-                                   StringEncoding.format(this.iter.value())));
+                LOG.info(String.format("%s=%s",
```
Comment (seagle-yuan) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1067]: Would you remove "String.format"

such as 

https://github.com/apache/incubator-hugegraph/pull/1838#discussion_r855050621

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBOptions.java [hunk 4]
```diff
@@ -168,8 +168,8 @@ public static synchronized RocksDBOptions instance() {
     public static final ConfigOption<Integer> MAX_BG_JOBS =
             new ConfigOption<>(
                     "rocksdb.max_background_jobs",
-                    "Maximum number of concurrent background jobs," +
-                            " including flushes and compactions.",
+                    "Maximum number of concurrent background jobs, " +
+                          "including flushes and compactions.",
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBOptions.java:172]: align with "Maximum number..."

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java [hunk 5]
```diff
@@ -1059,13 +1059,12 @@ private boolean match(int expected) {
...
-            LOG.info(">>>> scan from " + this.table + ": "  +
+            LOG.info(">>>> scan from {}: {}{}", this.table,
                     (this.keyBegin == null ? "*" :
-                            StringEncoding.format(this.keyBegin)) +
+                            StringEncoding.format(this.keyBegin)),
                     (this.iter.isValid() ? "" : " - No data"));
             for (; this.iter.isValid(); this.iter.next()) {
-                LOG.info(StringEncoding.format(this.iter.key()) +
-                        "=" +
+                LOG.info("{}={}",StringEncoding.format(this.iter.key()),
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1067]:
```java
LOG.info("{}={}",
         StringEncoding.format(this.iter.key()),
         StringEncoding.format(this.iter.value()));
```

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java [hunk 6]
```diff
@@ -1059,13 +1059,12 @@ private boolean match(int expected) {
         @SuppressWarnings("unused")
         private void dump() {
             this.seek();
-            LOG.info(">>>> scan from " + this.table + ": "  +
+            LOG.info(">>>> scan from {}: {}{}", this.table,
                     (this.keyBegin == null ? "*" :
-                            StringEncoding.format(this.keyBegin)) +
+                            StringEncoding.format(this.keyBegin)),
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1064]:
```java
LOG.info(">>>> scan from {}: {}{}", 
         this.table,
         this.keyBegin == null ? "*" : StringEncoding.format(this.keyBegin),
         this.iter.isValid() ? "" : " - No data"));
```

**Review by javeme — DISMISSED** [review]

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java [hunk 7]
```diff
@@ -1058,14 +1059,14 @@ private boolean match(int expected) {
...
         private void dump() {
             this.seek();
-            System.out.println(">>>> scan from " + this.table + ": "  +
-                               (this.keyBegin == null ? "*" :
-                                StringEncoding.format(this.keyBegin)) +
-                               (this.iter.isValid() ? "" : " - No data"));
+            LOG.info(">>>> scan from {}: {}{}", 
+                    this.table,
+                    this.keyBegin == null ? "*" : StringEncoding.format(this.keyBegin),
+                    this.iter.isValid() ? "" : " - No data"));
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1065]: `" - No data"))` => `" - No data")`
there is an error: `/home/runner/work/incubator-hugegraph/incubator-hugegraph/hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1065:61: expecting RCURLY, found ')' -> [Help 1]`

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java [hunk 8]
```diff
@@ -1058,14 +1059,14 @@ private boolean match(int expected) {
         @SuppressWarnings("unused")
         private void dump() {
             this.seek();
-            System.out.println(">>>> scan from " + this.table + ": "  +
-                               (this.keyBegin == null ? "*" :
-                                StringEncoding.format(this.keyBegin)) +
-                               (this.iter.isValid() ? "" : " - No data"));
+            LOG.info(">>>> scan from {}: {}{}", 
+                    this.table,
```
Comment (javeme) [inline hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1063]: align with `">>>>`?
code style: https://github.com/apache/incubator-hugegraph-doc/wiki/HugeGraph%E4%BB%A3%E7%A0%81%E9%A3%8E%E6%A0%BC%E6%8C%87%E5%8D%97

File: hugegraph-scylladb/src/main/java/com/baidu/hugegraph/backend/store/scylladb/ScyllaDBTablesWithMV.java [hunk 9]
```diff
@@ -230,24 +230,24 @@ protected void dropTable(CassandraSessionPool.Session session) {
...
     }
 
     public static class Edge extends CassandraTables.Edge {
 
-        private final String MV_LABEL2EDGE = mvLabelTable(this.table());
+        private final String mvLabel2Edge = mvLabelTable(this.table());
 
-        private final String LABEL = CassandraTable.formatKey(HugeKeys.LABEL);
-        private final List<String> KEYS = this.idColumnName().stream()
+        private static final String LABEL = CassandraTable.formatKey(HugeKeys.LABEL);
```
Comment (seagle-yuan) [inline hugegraph-scylladb/src/main/java/com/baidu/hugegraph/backend/store/scylladb/ScyllaDBTablesWithMV.java:243]: need static?

Comment (z7658329) [reply hugegraph-scylladb/src/main/java/com/baidu/hugegraph/backend/store/scylladb/ScyllaDBTablesWithMV.java:243]: 定义静态变量编译会报错，无法访问成员变量

File: hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java (hunk 8 above)
Comment (z7658329) [reply hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java]: image

File: hugegraph-scylladb/src/main/java/com/baidu/hugegraph/backend/store/scylladb/ScyllaDBTablesWithMV.java [hunk 10]
```diff
@@ -263,15 +263,15 @@ protected void createIndex(CassandraSessionPool.Session session,
                          "  SELECT * FROM %s " +
                          "  WHERE %s IS NOT NULL AND %s IS NOT NULL " +
                          "  PRIMARY KEY(%s, %s)",
-                         MV_LABEL2EDGE, this.table(),
-                         this.LABEL, this.PRKEYS_NN,
-                         this.LABEL, this.PRKEYS);
+                    mvLabel2Edge, this.table(),
```
Comment (javeme) [inline hugegraph-scylladb/src/main/java/com/baidu/hugegraph/backend/store/scylladb/ScyllaDBTablesWithMV.java:266]: align with` "`

File: hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java [hunk 11]
```diff
@@ -107,12 +107,12 @@ public Number queryNumber(CassandraSessionPool.Session session,
             statement.setReadTimeoutMillis(timeout * 1000);
             return session.query(statement);
         }, (q, rs) -> {
-            Row row = rs.one();
-            if (row == null) {
-                return IteratorUtils.of(aggregate.defaultValue());
-            }
-            return IteratorUtils.of(row.getLong(0));
-        });
+                Row row = rs.one();
```
Comment (javeme) [inline hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraTable.java:110]: according to the latest conclusion, we can keep the original style lambda alignment

**Review by javeme — APPROVED** [review]

**Review by coderzc — APPROVED** [review]

**Pull request merged**